data/human/paradise world jobs.txt

```diff
@@ -521,3 +521,743 @@ mission "Paradise Job: Wilderness Retreat"
+	word
+		"an apartment"
+		"a house"
+	word
+		" on <planet>."
+
+
+phrase "martini evac dialog plural"
+	word
+		"Throughout the trip"
+		"While on board"
+		"During their time on board"
+		"Between hyperjumps"
+	word
+		", your passengers from Martini have been "
+	phrase
+		"martini evac 2"
+
+phrase "martini evac 2"
+	word
+		"accusing"
+		"repeatedly accusing"
+		"denouncing"
+	word
+		" "
+	word
+		"the Free Worlds"
+		"the Southern Defense Pact"
+		"the Southern Pact"
+		"'those ${insults for southerners}'"
+		"'them ${insults for southerners}'"
+		"'${insults for southerners}'"
+	word
+		" for the "
+	word
+		"nuclear bombings"
+		"destruction"
+		"devastation"
+	word
+		" on Martini. The group hands you <payment> and head off for their"
+	word
+		" " 4
+		" well-kept "
+		" expensive "
+		" opulent "
+		" exorbitant "
+	word
+		"penthouses"
+		"apartments"
+		"houses"
+		"villas"
+	word
+		" and "
+	word
+		"estates"
+		"condos"
+		"mansions"
+		"manors"
+	word
+		" on <planet>."
+
+phrase "martini evac 2"
+	word
+		"telling you"
+		"letting you know"
+		"saying"
+		"informing you"
+	word
+		" "
+	word
+		"of the desperation of the people on Martini"
+		"of the people they knew who died in the blasts"
+		"of the dire conditions that the people are facing on Martini"
+		"of the deteriorating state on the ground on Martini"
+		"of the destruction on Martini"
+		"that some of them haven't heard back from their relatives on Martini"
+	word
+		". They "
+	word
+		"implore you to help out further in the relief efforts,"
+		"thank you for bringing them off-planet,"
+	word
+		" hand you <payment> and head off for their "
+	word
+		"apartments"
+		"houses"
+		"apartments and houses"
+	word
+		" on <planet>."
+
+phrase "insults for southerners"
+	word
+		"bloody"
+		"damn"
+		"jumpy"
+		"upstart"
+		"whining"
+		"whingeing"
+		"stupid"
+		"seditious"
+		"treasonous"
+		"betraying"
+		"treacherous"
+		"deceptive"
+		"terrorizing"
+		"greedy"
+	word
+		" "
+	word
+		"southerners"
+		"collectivists from Bourne"
+		"subversives from the Rim"
+		"insurrectionists from the Rim"
+		"charity-cases from the South"
+		"rocketeers from Solace"
+		"pirates from Greenrock"
+		"unlawfuls from Thule"
+		"gunrunners from Rust"
+
+
+
+mission "Martini Relief (Medical) [0]"
+	name "Disaster relief to Martini"
+	description "Due to the recent nuclear bombings, bring <cargo> to <planet> to help treat the wounded. Payment is <payment>."
+	job
+	repeat
+	cargo Medical 5 25
+	clearance "When you explain to the spaceport controller that you are carrying a vital supply of <commodity> to help treat the wounded, they give you the clearance to land."
+	to offer
+		has "event: war begins"
+		not "event: initial deployment 2"
+		random < 25
+	source
+		near Pollux 2 10
+	destination Martini
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 32000 50
+		dialog phrase "generic cargo delivery payment"
+
+
+mission "Martini Relief (Medical) [1]"
```

Comment:
The only difference I see between these two are the payment. Why do you need both?

Reply:
There'll be a lot of people injured, blinded, suffering from radiation sickness, etc., and I wanted to emphasize that in the jobs.

Reply:
You could just change the random chance to make it pop up more often.

Reply:
But there would only be one mission that would be offered at a time.